probably/pgcl/check.py

```diff
@@ -238,8 +237,7 @@ def _get_distribution_type(prog: Program, expr: Expr, check: bool = True) -> Uni
         if isinstance(expr.sampling_dist, get_args(DistrExpr)):
             return _get_distribution_type(prog, expr.sampling_dist, check)
         else:
-            print(Style.OKRED + "Type Checking is not accurate anymore." + Style.RESET)
-            return check_expression(prog, expr.sampling_dist)
+            return get_type(prog, expr.sampling_dist, check)
```

Comment:
I still don't think that's correct, but at least it's typesafe

Reply:
Yeah, I'm not sure that is correct. @LKlinke what's the return type of an iid-expression with a PGF expression inside?